### ISSUE TITLE
On-prem 2015 Update 2.1 - No widgets appear in list after install

### ISSUE THREAD

**Issue by sumo300** [jmezach/NewRelicDashboardWidgets#4]
I've installed this in a 2015 Update 2.1 on-prem version of TFS.  However, when I look at the list of available widgets for the dashboard, nothing from this set appears.  Note that I have installed it into the collection I am in.

**Comment by jmezach** [comment]
Unfortunately this is a limitation of on-premise TFS at the moment. You cab install extensions that add dashboard widgets but you can't actually use them.

Luckily this should be coming in TFS 2015 Update 3 according to [this](https://www.visualstudio.com/en-us/news/release-archive-vso.aspx). I believe that Update 3 should be coming somewhere this month.

I'll leave this issue open as a reminder to test out these widgets on on-prem when Update 3 is out and update it accordingly.

**Comment by jmezach** [comment]
In fact I just saw that a release candidate for TFS 2015 Update 3 has been released including support for dashboard extensions. You can read all the details [here](https://blogs.msdn.microsoft.com/bharry/2016/06/07/team-foundation-server-2015-update-3-rc-is-available/).

I'll give it a spin and see if everything works there.

**Comment by jmezach** [comment]
I just installed TFS 2015 Update 3 RC and installed the widgets from the marketplace and they seem to work just fine:

![image](https://cloud.githubusercontent.com/assets/1225489/15892692/69b5be32-2d7b-11e6-83d0-03c2d5f2a9af.png)

I did notice an issue when I was using Internet Explorer, but I think that might be a separate issue. Closing this issue for now. Let me know if you run into any problems after updating to Update 3.
